### ISSUE TITLE
Event "closed" fires when opening and closing of the datepicker

### ISSUE THREAD

**Issue by focussing** [charliekassel/vuejs-datepicker#280]
`<datepicker v-model="startDate" :monday-first="true" input-class="input" placeholder="Selecteer..." v-on:closed="doSelection()"></datepicker>`

and

```
    methods: {
        doSelection: function() {
            console.log('event fired');
        },
    },
```

When the datepicker is clicked it opens and the console.log shows "event fired".
After selecting a date the console.log shows "event fired" again.

**Comment by focussing** [comment]
any news on this?

**Comment by mateo2181** [comment]
Try this: 
`<datepicker v-model="startDate" :monday-first="true" input-class="input" placeholder="Selecteer..." v-on:closed="doSelection"></datepicker>`

**Comment by Landen13** [comment]
Open action triggers both events: **closed** and **opened** (in this order).
Switching between datepicker modes (month, year, date) triggers **closed** action also.

**Comment by aphilippartd** [comment]
Same issue here. Any progress on this?

**Comment by charliekassel** [comment]
Yes this should be fixed.
Can you update to latest and confirm please?

**Comment by Landen13** [comment]
@aphilippartd it is fixed in 0.9.21+
